internal/url/url.go

```diff
@@ -7,10 +7,23 @@ import (
 	"strings"
 )
 
+const (
+	urlUnescape   = true
+	noUrlUnescape = false
+)
+
 // ParseQuery parses the URL-encoded query string and returns the corresponding map.
 // It takes separators as parameter, for example: & or ; or &;
-// It returns error if the query string is malformed.
 func ParseQuery(query string, separator byte) map[string][]string {
+	return doParseQuery(query, separator, urlUnescape)
```

Comment:
if constants are private and since the upstream method is descriptive enough I think it is OK to drop the constants.

Reply:
I kept the constants for readability purposes, but I'm good either way

Reply:
I'd rather remove them.